### ISSUE TITLE
Enduro freezing / going blank after some time

### ISSUE THREAD

**Issue by skooterblog** [MiSTer-devel/Atari2600_MiSTer#35]
I noticed that Enduro is freezing or going blank after some time. It takes several minutes, sometimes it will last for more than an hour, but it will eventually happen. To reproduce, just launch the game and let it run its demo.

**Comment by skooterblog** [comment]
For example. In this time, it happened after I played for about an hour. I suddenly got a red screen:
https://youtu.be/xrPMt56TScg

**Comment by PanterACFH** [comment]
Yeah, mine did the same. I left it running when i went to work, came home to a black screen. Pressing F12 brought the MiSTer menu up though, but the game is dead.

**Comment by skooterblog** [comment]
@PanterACFH Thank you for testing it.

Sorry for the false alarm. I just tested and it also happens on the real hardware. After a while, the screen goes blank, if reset is pressed it comes back. It also happened on H.E.R.O. It didn't happen on Snoopy vs. the Red Baron. Maybe it's an Activision thing to protect against screen burn-in or something like that.

The red screen I got yesterday is probably unrelated, or maybe it happened because I disabled the framebuffer (vsync_adjust=2) to play Enduro.

I'll test further, but this doesn't seem to be a MiSTer issue, so I'm closing this issue.
